### PR TITLE
1) Fixed bug with clamp range in scale estimation.

### PR DESCRIPTION
**Pull request by andreyanufr**
### Changes

Fixed bug with clamp ranges in scale estimation.

### Reason for changes

bug

### Tests

Unit test.

### CODE REVIEW

**Comment by andreyanufr** [comment]
@alexsu52 post_training_weight_compression 82

**Review by alexsu52 — APPROVED** [review]
LGTM

**Review by l-bat — APPROVED** [review]

File: nncf/quantization/algorithms/weight_compression/weight_lowering.py [hunk 1]
```diff
@@ -295,7 +300,11 @@ def calculate_quantized_weight(
     level_low = 0 if asym_quant else -(2 ** (num_bits - 1))
     level_high = 2**num_bits - 1 if asym_quant else 2 ** (num_bits - 1) - 1
 
-    compressed_weights = weight / scale
+    if invert_scale:
```
Comment (alexsu52) [inline nncf/quantization/algorithms/weight_compression/weight_lowering.py:303]: What's your plan? I suggest checking the possibility of making `invert_scale=True` by default.

**Review by alexsu52 — CHANGES_REQUESTED** [review]

File: nncf/quantization/algorithms/weight_compression/weight_lowering.py (hunk 1 above)
Comment (andreyanufr) [reply nncf/quantization/algorithms/weight_compression/weight_lowering.py:303]: Yes, but in the next PR.

Comment (alexsu52) [reply nncf/quantization/algorithms/weight_compression/weight_lowering.py:303]: Please, submit a ticket for this task.

**Review by alexsu52 — APPROVED** [review]

**Pull request merged**